### PR TITLE
Add VLMPipeline to root README

### PR DESCRIPTION
**Pull request by Wovchena**
And fix links after <deteails> tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,9 +98,54 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Create+a
...
+    std::string model_path = argv[1];
+    ov::genai::VLMPipeline pipe(model_path, "CPU");
+    ov::Tensor rgb = utils::load_image(argv[2]);
+    std::cout << pipe.generate(prompt, ov::genai::image(rgb)) << '\n';
+}
+```
+
+### Sample notebooks using this API
+
+See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Visual-language+assistant+with+MiniCPM-V2+and+OpenVINO)
```
Comment (eaidova) [inline README.md:141]: @yatarkan  could you please advise how we can better build filter for github pages for viewing genai notebooks based on use cases? It is ok if something will required to add in notebooks meta and not blocker for this PR

**Review by eaidova — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -97,9 +98,54 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Create+a
...
+```sh
+optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code MiniCPM-V-2_6
+```
+
+### Run generation using VLMPipeline API in Python
+
+```python
+import openvino_genai as ov_genai
+#Will run model on CPU, GPU
+pipe = ov_genai.VLMPipeline("./MiniCPM-V-2_6/", "CPU")
```
Comment (ilya-lavrenov) [inline README.md:117]: the comment looks confusing. You create pipeline on CPU, while comments states it will run on CPU / GPU

**Review by ilya-lavrenov — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eaidova) [reply README.md:117]: maybe better to add device variable before pipeline and pass it instead of string?

File: README.md [hunk 3]
```diff
@@ -97,9 +98,54 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Create+a
...
+rgb = read_image("cat.jpg")
+print(pipe.generate(prompt, image=rgb))
```
Comment (eaidova) [inline README.md:119]: do we need explicitly add max_new_tokens like in llm example above?

Comment (Wovchena) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (Wovchena) [reply README.md:117]: Updated

**Review by ilya-lavrenov — APPROVED** [review]

**Review by yatarkan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eaidova) [reply README.md:145]: yes, that is what we need

**Pull request merged**